go/test/endtoend/cluster/cluster_process.go

```diff
@@ -246,6 +245,11 @@ func (cluster *LocalProcessCluster) StartTopo() (err error) {
 		cluster.VtctlProcess.LogDir = cluster.TmpDirectory
 	}
 
+	if err = cluster.TopoProcess.OpenServer(*topoFlavor, cluster.VtctlProcess.TopoGlobalRoot); err != nil {
```

Comment:
```suggestion
	if err = cluster.TopoProcess.OpenServer(*topoFlavor, cluster.VtctldClientProcess.TopoGlobalRoot); err != nil {
```

Reply:
Does not exist (note the original was `VtctlProcess` _not_ `VtctlclientProcess`)